### ISSUE TITLE
Incorrect content rendering when inserting HTML

### ISSUE THREAD

**Issue by chdenat** [wp-shortcake/shortcake#312]
I've just installed  the 0.3 release and got an issue with inner content.

Here's my piece of cake (upgraded to take into account the inner_content.

```
    if ( function_exists( 'shortcode_ui_register_for_shortcode' ) ) {
        shortcode_ui_register_for_shortcode(
            'my-shortcode',
            array(
                'label'         => 'My shortcode',
                'listItemImage' => 'dashicons-admin-page',
                inner_content=> array(
                    'label' => 'Text',
                ),
                'attrs'         => array(
                    array(
                        'label'   => 'Alignement',
                        'attr'    => 'float',
                        'type'    => 'select',
                        'options' => array(

     bla bla...
```

Then  I create a new shortcode entry in my post using the shortcake ui and put 

```
    <h2>HELLO</H2>
```

In the shortcode edit menu, the preview is nice, my shortcode content is well formatted, my css classes are applied...So good. 

But in my post edit the rendering is broken and I see (HELLO string is well formatted, in H2 in this case)

```
[my-shortcode]
HELLO
[/my-shortcode]
```

And I can not edit my shortcode anymore through the shortcake UI.

Did I make something wrong  or is it real issue  ?

Thanks

**Comment by danielbachhuber** [comment]
> Did I make something wrong or is it real issue ?

You've registered the new UI appropriately. There's a bug in core though: https://core.trac.wordpress.org/ticket/32078

**Comment by hirejordansmith** [comment]
I believe I'm having the same or similar issue on my sites? Essentially I can't get this to function correctly using v0.3.0 so went to my dev server to play with it.

I set up the default functionality for this with the pull quote - https://gist.github.com/Mte90/cb8a0e37565d219062ec

And Shortcake 0.3.0 installed (only plugin activated)

Running Twenty Fifteen Theme - WP 4.2.2

When I add the pullquote, I'm able to edit and all but the content doesn't even show up in the editor or on the front end - http://grab.by/HZGG - http://grab.by/HZGM (http://www.hjsdev.com/shortcake-ui/)

This was working find in 0.2.3, has stopped working correctly once updated.

One other thing I noticed while playing around with both versions (0.2.3 and 0.3.0) is that the actual shortcode structure being output is different.

0.2.3 outputs shortcodes like this - [pullquote source="Jordan"]Quote Content[/pullquote]

and 

0.3.0 outputs shortcodes like this - [pullquote content="Quote Content" source="Jordan"]

If I have shortcodes in place from 0.2.3 and update, they still display on the front end but if I add new shortcodes it doesn't display the content in the editor or the front end.

http://www.hjsdev.com/shortcake-ui/
http://grab.by/HZJ2 - the output (top is 0.2.3, bottom is 0.3.0)
http://grab.by/HZJ6 - visual editor view
http://grab.by/HZJe - Front End output

Is anyone else having this issue with 0.3.0? If so, is this the issue that is going to be fixed in the next update?

Love the plugin. Thanks for the hard work! I know support can be grueling at time.

**Comment by danielbachhuber** [comment]
> I believe I'm having the same or similar issue on my sites?

Nope. As [documented in the release notes](http://fusion.net/story/126834/introducing-shortcake-v0-3-0-butter/), we've changed `content` to `inner_content`. You need to register your UI differently.

**Comment by hirejordansmith** [comment]
Ah Crap, sorry man. I never saw those notes. I've signed up for updates so I stay in touch. Thanks!
